src/Avalonia.Themes.Fluent/Controls/ColorSpectrum.xaml

```diff
@@ -0,0 +1,138 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True"
+        xmlns:converters="using:Avalonia.Controls.Converters">
+
+  <Design.PreviewWith>
+    <Border Padding="20">
+      <ColorSpectrum Width="256" Height="256" />
+    </Border>
+  </Design.PreviewWith>
+
+  <Styles.Resources>
+    <converters:EnumValueEqualsConverter x:Key="EnumValueEquals" />
+  </Styles.Resources>
+
+  <Style Selector="ColorSpectrum">
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate>
+          <Grid x:Name="PART_LayoutRoot">
+            <Grid x:Name="PART_SizingGrid"
+                  HorizontalAlignment="Center"
+                  VerticalAlignment="Center">
+              <Grid.Clip>
+                <RectangleGeometry />
+              </Grid.Clip>
```

Comment:
In Avalonia and WPF you usually can just use ClipToBounds="True".

Reply:
Upd: oh, I see. There is custom clip rectangle set in code behind, not necessary the same as bounds.

Reply:
Yea, that's the way it functions in UWP. I'm not sure why they couldn't just change the size of the "PART_SizingGrid" though (which is inside "PART_LayoutRoot" that holds the actual control dimensions). Doing that would avoid these special clipping geometries and allow just using `ClipToBounds`. I didn't look at this in detail though, it was just ported.

Reply:
I'm going to mark this one as resolved. I previously marked it as unresolved since I ended up making the changes you suggested. I'm assuming you have read this by now so no point keeping it open.